packages/rxjs/integration/import/fixtures/browser/index.mjs

```diff
@@ -0,0 +1,61 @@
+import puppeteer from 'puppeteer';
+import { join, dirname } from 'node:path';
+import { fileURLToPath } from 'node:url';
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = dirname(__filename);
+
+async function main() {
```

Comment:
This integration test will load up `index.html` (in the same directory) and check `<script type="module"></script>` web module imports.

See `browser-test.js` for the other half of these tests.